.env

```diff
@@ -0,0 +1,8 @@
+MYSQL_DATASOURCE_URL=jdbc:mysql://db:3306/book_store?serverTimeZone=UTC
+MYSQL_DATASOURCE_USERNAME=root
+MYSQL_DATASOURCE_PASSWORD=admin
+SPRING_LOCAL_PORT=8081
+SPRING_DOCKER_PORT=8080
+MYSQL_DATABASE=book_store
+MYSQL_PORT=3307
+DOCKER_MYSQL_PORT=3306
```

Comment:
add empty lines for all files (or it's some bug on github?)

Reply:
this is probably a bug